src/store/LastDigitStatsStore.js

```diff
@@ -56,7 +56,7 @@ export default class LastDigitStatsStore {
         return this.mainStore.state.shouldMinimiseLastDigits;
     }
 
-    @action.bound async updateLastDigitStats(response = {}) {
+    @action.bound async updateLastDigitStats() {
```

Comment:
```suggestion
    @action.bound async updateLastDigitStats(response) {
```
Removing the default assignment alone should fix the problem.